website/components/remote-plugin-docs/utils/resolve-nav-data.js

```diff
@@ -139,7 +139,7 @@ async function resolvePluginEntryDocs(pluginConfigEntry) {
     // Process into a NavLeaf, with a remoteFile attribute
     const dirs = path.dirname(filePath).split('/')
     const dirUrl = dirs.slice(2).join('/')
-    const basename = path.basename(filePath)
+    const basename = path.basename(filePath, path.extname(filePath))
```

Comment:
I'd assumed `basename` left out the extension by default, it doesn't 😬 Have corrected to ensure the extension isn't present 👍

Reply:
Nice!